### PR TITLE
DOC linkfield accesibility improvements

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-linkfield/issues/245

### CODE REVIEW

File: en/08_Changelogs/5.3.0.md
Comment (ChloeSartorelli) [inline en/08_Changelogs/5.3.0.md]:
```suggestion
- `silverstripe/linkfield` now has improved accessibility support for screen readers and keyboard navigation. Focus states have also been made consistent between keyboard and mouse interaction.
```
The latter part of that is really just a subjective rewording so feel free to take or leave it - the main thing here is the typo for "accessibility"

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/5.3.0.md
Comment (emteknetnz) [reply en/08_Changelogs/5.3.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge after the code is merged

**Pull request merged**
